### DISCUSSION TITLE
Force PTR / Better QueryLog Filtering and Statistic Filtering / DNS Domains Infos

### DISCUSSION THREAD

**Discussion by whyisthisbroken** [AdguardTeam/AdGuardHome#5673] [Ideas]
Hey Guys,

i know it´s been annoying and maybe 1000x answered - but i think the community waits also for these features and squint up a little to pihole....

1. Force PTR request 
I don't know how the PTR in AGH works, only on start / occasionally or something between. sometimes it works, sometime not, a button to force a PTR reverse request would be nice to test the server that AGH should use for PTR request...
i have so many devices where AGH don't show a Name - but nslookup has no problems with. 
yes - we have a button for testing the DNS Server settings at all, but i think we should get an answer more then: "Working".
it´s a DNS - Server.

Server so why not detailed answers ?

2. Better Querylog and Statistic Filtering
We need a better statistic in AGH - like pihole.
What we need ? 
Filtering baes on Blocklist - wich has more Queryblocks? So, i can remove lists that doesn't was used and save ram.
Filtering based on DNS Upstream Server - nice to see wich is the fastest and has more responses
Filtering by Query Type and answers - A or AAAA - cache or not ?

So, this should also be seen in the statistic on the Dashboard - this are essential information for the user.

3. DNS Domain Infos
The DNS Blocklist is built by AGH in the Background when updating - i hope that AGH recognize double entries etc. ?
So why i don't see how many Domains are used by AGH, or how much RAM AGH use.

I´ve seen a glimpse in the BETA Dashboard that was removed not a long time ago where this feature was integrated - RAM Usage. Domains etc... i don't understand why this project was given up :( 
It was a bit closer to that what the community want since a long time.... maybe a dev could give me some answers :)

**Comment by king7778** [comment]
ewdssd
